plugins/community/google-gnostic-openapi/v0.7.0/buf.plugin.yaml

```
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/community/google-gnostic-openapi
```

```suggestion
name: buf.build/community/google-openapi
```

This alternative drops the "gnostic" from the name, which would be more in line with our existing nomenclature conventions of: `{owner}-{name}`

Where the repository is the upstream GitHub owner and the name is usually the name of the plugin minus `protoc-gen-` prefix.

However, for this particular one having the name "gnostic" in the name makes the most sense given the project's background. For some context:

- https://github.com/grpc-ecosystem/grpc-gateway/issues/441
- https://github.com/google/gnostic/issues/340

Although this is parked under github.com/gnostic organization,

---

I prefer having the `gnostic` in the name too.